### ISSUE TITLE
Update documentation for wallet 0.54

### ISSUE THREAD

**Issue by edd** [vegaprotocol/documentation#179]
See: https://github.com/vegaprotocol/vega/issues/5766

# tl;dr
- [vegaprotocol/go-wallet](https://github.com/vegaprotocol/go-wallet) is being deprecated. 
- https://github.com/vegaprotocol/vega/issues/5766 will push people to upgrade at the appropriate time
- The wallet that works for 0.54 will not work with networks below 0.54
- The wallet that works for older networks will not work on 0.54

# Tasks
- [x] ~Update all commands to the new format (probably `vega wallet` instead of `vegawallet`~ Not required
- [x] Prune old wallet documentation (may need to be a separate issue so that it happens at a more appropriate time
- [x] Update tools links to point to the proper repo

**Comment by edd** [comment]
Created for future reference, marked as blocked while 0.54 still isn't out
